misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/util/CodeCoverageUtils.java

```diff
@@ -140,6 +140,8 @@ private static boolean isRequiredFile(String path, String orgName, boolean enabl
             return false;
         } else if (enableIncludesFilter && !isIncluded(path, includesInCoverage)) {
             return false;
+        } else if (path.contains("-generated_")) {
```

Comment:
If user has a file with this name, the feature would break. I think we need to think of a different approach

Reply:
Also, we need to ignore the changes done by the code modifiers